imperial_coldfront_plugin/forms.py

```diff
@@ -120,6 +122,15 @@ class RDFAllocationForm(forms.Form):
         disabled=False,
     )
 
+    def clean_dart_id(self) -> str:
```

Comment:
This is one of those methods that are magically called by Django, based on the variable name, right? I'm asking because I don't see it being used anywhere - at least not explicitly.

Reply:
Yeah that's right. Form magic.